sos/plugins/cs.py

```diff
@@ -1,128 +1,140 @@
-                "/opt/redhat-cs/slapd-*/logs/access",
-                "/opt/redhat-cs/slapd-*/logs/errors",
-                "/opt/redhat-cs/slapd-*/config/dse.ldif",
-                "/opt/redhat-cs/cert-*/errors",
-                "/opt/redhat-cs/cert-*/config/CS.cfg",
-                "/opt/redhat-cs/cert-*/access",
-                "/opt/redhat-cs/cert-*/errors",
-                "/opt/redhat-cs/cert-*/system",
-                "/opt/redhat-cs/cert-*/transactions",
-                "/opt/redhat-cs/cert-*/debug",
-                "/opt/redhat-cs/cert-*/tps-debug.log"
+                '/opt/redhat-cs/slapd-*/logs/access',
+                '/opt/redhat-cs/slapd-*/logs/errors',
+                '/opt/redhat-cs/slapd-*/config/dse.ldif',
+                '/opt/redhat-cs/cert-*/errors',
+                '/opt/redhat-cs/cert-*/config/CS.cfg',
+                '/opt/redhat-cs/cert-*/access',
+                '/opt/redhat-cs/cert-*/errors',
+                '/opt/redhat-cs/cert-*/system',
+                '/opt/redhat-cs/cert-*/transactions',
+                '/opt/redhat-cs/cert-*/debug',
+                '/opt/redhat-cs/cert-*/tps-debug.log'
             ])
         if csversion == 73:
             self.add_copy_spec([
-                "/var/lib/rhpki-*/conf/*cfg*",
-                "/var/lib/rhpki-*/conf/*.ldif",
-                "/var/lib/rhpki-*/logs/debug",
-                "/var/lib/rhpki-*/logs/catalina.*",
-                "/var/lib/rhpki-*/logs/ra-debug.log",
-                "/var/lib/rhpki-*/logs/transactions",
-                "/var/lib/rhpki-*/logs/system"
-            ])
-        if csversion in (73, 8):
-            self.add_copy_spec([
-                "/etc/dirsrv/slapd-*/dse.ldif",
-                "/var/log/dirsrv/slapd-*/access",
-                "/var/log/dirsrv/slapd-*/errors"
+                '/var/lib/rhpki-*/conf/*cfg*',
+                '/var/lib/rhpki-*/conf/*.ldif',
+                '/var/lib/rhpki-*/logs/debug',
+                '/var/lib/rhpki-*/logs/catalina.*',
+                '/var/lib/rhpki-*/logs/ra-debug.log',
+                '/var/lib/rhpki-*/logs/transactions',
+                '/var/lib/rhpki-*/logs/system'
             ])
         if csversion == 8:
             self.add_copy_spec([
-                "/etc/pki-*/CS.cfg",
-                "/var/lib/pki-*/conf/*cfg*",
-                "/var/log/pki-*/debug",
-                "/var/log/pki-*/catalina.*",
-                "/var/log/pki-*/ra-debug.log",
-                "/var/log/pki-*/transactions",
-                "/var/log/pki-*/system"
+                '/var/lib/pki-*/conf/*cfg*',
+                '/var/log/pki-*/debug',
+                '/var/log/pki-*/catalina.*',
+                '/var/log/pki-*/localhost.*',
+                '/var/log/pki-*/manager.*',
+                '/var/log/pki-*/ra-debug.log',
+                '/var/log/pki-*/transactions',
+                '/var/log/pki-*/selftests.log',
+                '/var/log/pki-*/system'
```

Comment:
There's a lot of log data mixed up with configuration here: generally in `sos` we try to separate those, so that for e.g. size limits can be applied to log data. If we are going to update this plugin for the next release it would be good to include that.